packages/s3-store/index.ts

```diff
@@ -70,7 +70,8 @@ export class S3Store extends DataStore {
   private client: S3
   private preferredPartSize: number
   public maxMultipartParts = 10_000 as const
-  public minPartSize = 5_242_880 as const // 5MB
+  public minPartSize = 5_242_880 as const // 5MiB
+  public maxUploadSize = 5_497_558_138_880 as const // 5TiB
```

Comment:
We are confusing the max total size vs the max part size here I think

```suggestion
  public maxPartSize = 5_368_709_120  as const // 5GiB
```